client/.eslintrc.js

```diff
@@ -23,6 +23,11 @@ module.exports = {
   env: {
     browser: true,
     es6: true,
+    commonjs: true
```

Comment:
I'm not sure what the scoping means and how it relates to us using ES modules, but I'd just want to avoid exposing e.g. the `require` function when an import should be used. Not a big deal, though.